src/backend/base/langflow/services/settings/base.py

```diff
@@ -70,6 +70,8 @@ class Settings(BaseSettings):
     dev: bool = False
     """If True, Langflow will run in development mode."""
     database_url: str | None = None
+    """Database driver name async. If not provided, Langflow will use a SQLite driver async."""
+    database_driver_async: str | None = None
     """Database URL for Langflow. If not provided, Langflow will use a SQLite database."""
```

Comment:
It's still NOK. `database_url` and `database_driver_async` docstrings must be switched.